templates/pages/account/orders/invoice.html

```diff
@@ -100,12 +117,21 @@ <h2 class="InvoiceHeading">{{lang 'invoice.order_items'}}</h2>
                 </thead>
                 <tbody class="InvoiceItemList">
                 {{#each order.items}}
-                    <tr>
+                    <tr {{#if new_address}}class="NewAddress"{{/if}}>
                         {{#if ../order.has_multiple_shipping_addresses}}
                             <td class="ProductAddress">
                                 {{#each shipping_rows}}
+                                    {{#if is_pickup}}
+                                        <div><b>{{lang 'invoice.pickup'}}</b></div>
+                                    {{/if}}
+                                    {{#if is_shipping}}
+                                        <div><b>{{lang 'invoice.shipping'}}</b></div>
+                                    {{/if}}
+                                    {{#if is_digital}}
+                                        <div><b>{{lang 'invoice.digital'}}</b></div>
+                                    {{/if}}
```

Comment:
👏 Nice approach, you can work as product designer Willem 😄

Reply:
LOL